libmamba/src/api/install.cpp

```diff
@@ -444,7 +444,11 @@ namespace mamba
     {
         using Request = solver::Request;
 
-        request.jobs.reserve(request.jobs.size() + (!no_pin) * ctx.pinned_packages.size() + !no_py_pin);
+        LOG_WARNING << "DEBUG WARNING: no_py_pin = " << (no_py_pin ? "true" : "false");
```

Comment:
We can remove this now I guess (or if you think this can be useful, use `LOG_DEBUG` instead?).

Reply:
ah yes I added this to check something and forgot to remove it, will do